_episodes/05-null.md

```diff
@@ -226,6 +226,16 @@ detail in [the next section]({{ site.github.url }}/06-agg/).
 >
 > to produce?
 > What does it actually produce?
+> > ## Solution
+> >
+> > The IN () function does not match NULL, and we only get results for the date '1927-02-08'.
```

Comment:
Perhaps it's even good to recap what IN really does at this point:
```suggestion
> > We only get a result for the value '1927-02-08', not for NULL. This is because `dated IN ('1927-02-08', NULL)` is just a shorthand for `dated = '1927-02-08' OR dated = NULL`, and we now know that the latter condition (`dated = NULL`) is never true. 
```

Reply:
We can get back to this in a separate issue